_jobs/2019-09-16-octobot.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: base
+
+name: "Octobot"
+hidden: false
+website: "https://www.octobot.io/careers"
+website_text: "Apply here"
+description: |
+
+Junior Developer
+
+We are proud of what we do and we are always on the look for people that feel the same and are seeking to apply their empathy, care, knowledge and expertise to create new products for a world-wide audience.
+---
```

Comment:
This one too.

![image](https://user-images.githubusercontent.com/7773256/65043606-d3f8f580-d920-11e9-82b2-f4cd968ba689.png)